docs/setup.md

```diff
@@ -11,26 +11,29 @@
 
 The **SOC Experience** solution pack depends on the following solution packs that are installed automatically &ndash; if not already installed.
 
-| Solution Pack Name | Purpose                                                  |
-|:-------------------|:---------------------------------------------------------|
-| SOAR Framework     |  Required for Incident Response modules                   |
-| SOC Simulator      |  Required for Scenario Module and SOC Simulator connector |
-| Phishing Email Response      | Required to demostrate investigation of Suspicious Emails |
-| Brute Force Attack Response    |  Required to demostrate investigation of Brute Force Attacks  |
-| MITRE ATT&CK Enrichment Framework | Required for MITRE Modules and MITRE connector |
-| C2 Malware Traffic Response | Required to demostrate investigation of C2 attacks |
-| Command and Control Response | Required to demostrate investigation of Command and Control attacks |
-| Lateral Movement and VPN Breach Response | Required to demostrate investigation of Lateral Movement and VPN Breach attacks |
-| FortiSIEM Essentials | Required for FortiSIEM integration in FortiSOAR |
-| Impossible Traveller Threat Response | Required to demostrate investigation of Concurrent Login type of attacks |
-| Data Leakage Threat Response | Required to demostrate investigation of Data Leakage Threat attacks |
-| SOC Overview Metrics | Required for SOC Management Widget and Dashboard |
-| Threat Intel Management | Required for Threat Intellgence Modules and Reports |
-| Continuous Delivery | Required to demostrate CICD |
+| Solution Pack Name                       | Version | Purpose                                                                          |

```

Comment:
@cs-abhishek-shukla As discussed, refer to the attached 
[SP Name.xls](https://github.com/fortinet-fortisoar/solution-pack-soc-experience/files/11850829/SP.Name.xls)
SP versions